### PR TITLE
i18n(de): translate components/using-components

### PR DESCRIPTION
**Pull request by trueberryless**
#### Description

Translation of `components/using-components.mdx` into German language.

### CODE REVIEW

**Comment by trueberryless** [comment]
Link fixed in https://github.com/withastro/starlight/pull/2458

File: docs/src/content/docs/de/components/using-components.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+<Icon name="open-book" />
+
+<CustomCard>
+	Komponenten können auch **verschachtelte Inhalte** enthalten.
+</CustomCard>
+```
+
+Da Starlight von Astro unterstützt wird, kannst du in deinen MDX-Dateien Unterstützung für Komponenten hinzufügen, die mit einem beliebigen [unterstützten UI-Framework (React, Preact, Svelte, Vue, Solid und Alpine)](https://docs.astro.build/de/core-concepts/framework-components/) erstellt wurden.
+Weitere Informationen über [Verwendung von Komponenten in MDX](https://docs.astro.build/en/guides/integrations-guide/mdx/#using-components-in-mdx) findest du in der Astro-Dokumentation.
```
Comment (HiDeoo) [inline docs/src/content/docs/de/components/using-components.mdx:36]:
```suggestion
Weitere Informationen über [Verwendung von Komponenten in MDX](https://docs.astro.build/de/guides/integrations-guide/mdx/#using-components-in-mdx) findest du in der Astro-Dokumentation.
```

File: docs/src/content/docs/de/components/using-components.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+# src/content/docs/example.mdoc
+title: Willkommen bei meiner Dokumentation
+---
+
+{% card title="Stars" icon="star" %}
+Sirius, Wega, Betelgeuse
+{% /card %}
+```
+
+Weitere Informationen zur Verwendung von Komponenten in Markdoc-Dateien findst du in der [Astro Markdoc Integrationsdokumentation](https://docs.astro.build/de/guides/integrations-guide/markdoc/#komponenten-rendern).
```
Comment (HiDeoo) [inline docs/src/content/docs/de/components/using-components.mdx:57]:
```suggestion
Weitere Informationen zur Verwendung von Komponenten in Markdoc-Dateien findst du in der [Astro Markdoc Integrationsdokumentation](https://docs.astro.build/de/guides/integrations-guide/markdoc/#render-components).
```

Fix anchor

File: docs/src/content/docs/de/components/using-components.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+---
+
+<div class="not-content">
+	<p>Die Standard-Inhaltsgestaltung von Starlight wirkt sich nicht aus.</p>
+</div>
+```
+
+## Komponenteneigenschaften
+
+Verwende den Typ [`ComponentProps`](https://docs.astro.build/de/guides/typescript/#componentprops-typ) aus `astro/types`, um auf die von einer Komponente akzeptierten `Props` zu verweisen, auch wenn sie nicht von der Komponente selbst exportiert werden.
```
Comment (HiDeoo) [inline docs/src/content/docs/de/components/using-components.mdx:83]:
```suggestion
Verwende den Typ [`ComponentProps`](https://docs.astro.build/de/guides/typescript/#componentprops-type) aus `astro/types`, um auf die von einer Komponente akzeptierten `Props` zu verweisen, auch wenn sie nicht von der Komponente selbst exportiert werden.
```

This section does not exist in the old version of the German page so let's keep the English anchor for now.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the new translation :star2:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://contribute.docs.astro.build/reviewers/reviewing-translations/). Left a few comments regarding links but except that, everything looks good to me and should be mergable once the link validation passes.

**Comment by trueberryless** [comment]
Thanks for the review, should be good to go!

**Review by HiDeoo — APPROVED** [review]
All links are now green, thanks for the updates :raised_hands:

**Pull request merged**